test/fancyflow_SUITE.erl

```diff
@@ -1,20 +1,15 @@
 -module(fancyflow_SUITE).
 -include_lib("common_test/include/ct.hrl").
 -include_lib("eunit/include/eunit.hrl").
--compile({parse_transform, fancyflow_trans}).
+-include("fancyflow.hrl").
 -compile(export_all).
 
 all() ->
-    [pipe, maybe, parallel,
-     pipe_trans, maybe_trans, parallel_trans,
-     mixed_trans].
+    [F || {F,1} <- ?MODULE:module_info(exports)]
+        -- [module_info, id, ok_id].
```

Comment:
please be explicit, go back to just listing tests. Right now this would explode if `init_per_suite` would be added and exported.